content/id/docs/concepts/workloads/pods/pod-preset.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Pod Preset
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+Halaman ini menyajikan gambaran umum tentang `Pod Preset`, yang merupakan objek untuk memasukkan informasi tertentu ke dalam `Pod` pada saat waktu penciptaan. Informasi dapat berupa _secret_, _volume_, _volume mount_, dan variabel _environment_.
+{{% /capture %}}
+
+{{% capture body %}}
+## Memahami **Pod Preset**
+---
+
+Sebuah `Pod Preset` adalah sebuah _resource_ API untuk memasukkan kebutuhan _runtime_ tambahan ke dalam sebuah _Pod_ pada saat waktu penciptaan. Kamu akan menggunakan _label selector_ untuk menunjuk _Pod-pod_ dimana _Pod Preset_ diterapkan.
```

Comment:
Pod doesn't need to be in italic

simply do:
Pod

same comment applies for the rest

Reply:
fixed in https://github.com/kubernetes/website/pull/14394/commits/bb4445855c8266473904a9566e5791344b91e11d